src/app/schemas/generation_template.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+from pydantic import BaseModel
+from pydantic.schema import datetime
+
+
+class GenerationTemplateBase(BaseModel):
```

Comment:
Can we add the class `Config` here to give an example how the data should look like? Like in the other schemas